src/main/java/org/fundacionjala/coding/persistence/Persistence.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.persistence;
+
+/**
+ * Class that is responsible for making multiplicative persistent calculations given an entry number.
+ */
+public class Persistence {
+    public static final int CONSTANT_MULTIPLICATION = 10;
+    public static final int CONSTANT_VERIFICATION = 9;
+
+    /**
+     * Method that performs the conversions of the persistent number.
+     *
+     * @param number is the persistence digital.
+     * @return cont with is the result..
+     */
+    public int persistenceDigit(int number) {
+        int counter = 0;
+        int result = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "result" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)